Sources/LiCore/Networking/SSOHandler.swift

```diff
@@ -25,72 +24,49 @@ class SSOHandler: RequestRetrier {
     private var isRefreshing = false
     private var requestsToRetry: [RequestRetryCompletion] = []
     func should(_ manager: SessionManager, retry request: Request, with error: Error, completion: @escaping RequestRetryCompletion) {
-        let data = request.delegate.data
-        do {
-            let error = try LiApiResponse.getLiBaseError(data: data)
-            if error.httpCode == 401 {
-                requestsToRetry.append(completion)
-                if !isRefreshing {
-                    refreshTokens { [weak self] succeeded, accessToken, refreshToken, expiresIn, error in
-                        guard let strongSelf = self else { return }
-                        strongSelf.lock.lock() ; defer { strongSelf.lock.unlock() }
-                        if error != nil {
-                            strongSelf.requestsToRetry.forEach { $0(false, 0.0) }
-                            strongSelf.requestsToRetry.removeAll()
-                            return
-                        }
-                        if let accessToken = accessToken, let refreshToken = refreshToken, let expiresIn = expiresIn {
-                            LiSDKManager.shared().authState.set(accessToken: accessToken)
-                            LiSDKManager.shared().authState.set(refreshToken: refreshToken)
-                            let currentDate = NSDate()
-                            let newDate = NSDate(timeInterval: expiresIn, since: currentDate as Date)
-                            LiSDKManager.shared().authState.set(expiryDate: newDate)
-                        }
-                        strongSelf.requestsToRetry.forEach { $0(succeeded, 0.0) }
+        lock.lock() ; defer { lock.unlock() }
+        if request.response?.statusCode == 401 {
```

Comment:
Please consider not hardcoding `401` and instead use an Http Status Code enumeration or create one.

Reply:
Ahh...already have it, will update.